libvirt/tests/src/virtual_network/qemu/netperf.py

```
@@ -0,0 +1,814 @@
+    netperf_version = params.get("netperf_version", "2.6.0")
+    client_path = "/tmp/netperf-%s/src/netperf" % netperf_version
+    server_path = "/tmp/netperf-%s/src/netserver" % netperf_version
+    get_status_flag = params.get("get_status_in_guest", "no") == "yes"
+    global _netserver_started
+    # Start netserver
+    if _netserver_started:
+        test.log.debug("Netserver already started.")
+    else:
+        error_context.context("Start Netserver on guest", test.log.info)
+        if params.get("os_type") == "windows":
+            timeout = float(params.get("timeout", "240"))
```

Ruff (0.14.10)

Local variable `timeout` is assigned to but never used

Remove assignment to unused variable `timeout`

(F841)